Add Rancher example multi-user doc (#127)
rancher/fleet-docs: docs/multi-user.md

:::

> ## Example Fleet standalone 

Reviewer: ```suggestion
## Example Fleet Standalone 
```

This makes sure, tenants can't interfere with GitRepo resources from other tenants, since they don't have access to their namespaces.

## Example Fleet in Rancher
 
When a new fleet workspace is created, a corresponding namespace with an identical name is automatically generated within the Rancher local cluster.

> For a user to see and deploy fleet resources in a specific workspace needs at least the following permissions:

Reviewer 2: Are we talking about permissions that the _user_ must have? If so, I find this a bit clearer:
```suggestion
For a user to see and deploy fleet resources in a specific workspace, they need at least the following permissions:
```

- list/get the `fleetworkspace` cluster-wide resource in the local cluster
- Permissions to create fleet resources (such as `bundles`, `gitrepos`, ...) in the backing namespace for the workspace in the local cluster. 

> Let's grants permissions to deploy fleet resources in the `project1` and `project2` fleet workspaces:

Reviewer 2: ```suggestion
Let's grant permissions to deploy fleet resources in the `project1` and `project2` fleet workspaces:
```

> - To create the `project1` and `project2` fleet workspaces, you can either do it in the  [Rancher UI](https://ranchermanager.docs.rancher.com/integrations-in-rancher/fleet/overview#accessing-fleet-in-the-rancher-ui) or use the following YAML resources:

Reviewer 2: nit (removing extra space):
```suggestion
- To create the `project1` and `project2` fleet workspaces, you can either do it in the [Rancher UI](https://ranchermanager.docs.rancher.com/integrations-in-rancher/fleet/overview#accessing-fleet-in-the-rancher-ui) or use the following YAML resources:
```
